Jatin Nansi notes that there's a miscount here which keeps entries that
also specify "restrict" from being matched properly.  RT#6889

Index: gssftp/ftpd/ftpd.c
===================================================================
--- gssftp/ftpd/ftpd.c	(revision 3308)
+++ gssftp/ftpd/ftpd.c	(working copy)
@@ -805,7 +805,7 @@
 			if (strcmp(line, name) == 0)
 			     return (1);
 			if (strncmp(line, name, strlen(name)) == 0) {
-			     int i = strlen(name) + 1;
+			     int i = strlen(name);
 			     
 			     /* Make sure foo doesn't match foobar */
 			     if (line[i] == '\0' || !isspace((int) line[i]))
